SWLOR.Game.Server/Feature/TrapDefinition/PitfallTrap.cs

```diff
@@ -0,0 +1,36 @@
+﻿using SWLOR.Game.Server.Core;
+using SWLOR.Game.Server.Core.NWNX;
+using SWLOR.Game.Server.Core.NWScript.Enum.Creature;
+
+namespace SWLOR.Game.Server.Feature
+{
+    public class PitfallTrap
+    {
+        /// <summary>
+        /// When this trap is triggered, it'll toss the players to the designated waypoint after a delay.
+        /// </summary>
+        [NWNEventHandler("OnTrapTriggered")]
```

Comment:
This is not going to work. Script names must all be lower case and no longer than 16 characters.